app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,47 +214,108 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
 
-You will be eligible for the IRP because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your job.
+You will be eligible for the first installment of IRP because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your job.
 
-### Non-eligibility
+### Eligibility requirements for IRP instalment 2 (£5,000)
 
-You will not be eligible for the international relocation payment if you are:
+To get the second instalment of the IRP, you must meet all the eligibility requirements set out in this section.
 
-* a UK citizen, including citizens of Wales, Scotland and Northern Ireland
+#### Teaching subject and school
 
-* an Irish citizen
+To be eligible for the second instalment of the IRP, you must:
 
-* a non-UK citizen who moved to England more than 3 months before the start of your job
+* have worked as a teacher for the previous year, that is, 3 consecutive terms
 
-* a non-UK citizen who has spent more than 90 days in the UK in the 12 months previous to the start of your job
+* be contracted to work as a teacher for at least another year
 
-* in England on a dependant visa linked to your partner or parent’s visa
+* be teaching in a subject and school which meets the requirements for the first instalment, as set out on this page
 
-* teaching only English language
+* have successfully applied for and received the first instalment of the IRP
 
-* employed on a contract lasting less than one year
+Your term time employment must be continuous, unless you [take time off work for statutory leave](https://www.gov.uk/browse/employing-people/time-off) take time off work for statutory leave, such as parental leave or sick leave.
+
+The Department for Education (DfE) will review time taken off for any other reasons on a case-by-case basis.
+
+You can move between eligible schools and remain eligible for the second instalment (see example 1).
+
+You can start midway through the academic year and as long as you complete 3 full terms, you will remain eligible for the second instalment (see example 2).
+
+Any term time employment gaps will make you ineligible for the second instalment (see example 3).
+
+#### Example 1
+You start your first eligible teaching job in England in September 2024 and successfully apply for the first instalment of the IRP. You work a full academic year of 3 terms, leaving the school in July 2025. You successfully apply for a new IRP eligible teaching job on a one-year contract starting in September 2025. You visit family overseas for the summer holidays and return to England to take up your teaching job and apply for the second instalment of the IRP. You are eligible for the second instalment of the IRP because you have been continuously employed as a teacher in eligible schools. 
```

Comment:
```suggestion
You start your first eligible teaching job in England in September 2024 and successfully apply for the first instalment of the IRP.  You work a full academic year of 3 terms, leaving the school in July 2025. 

You successfully apply for a new IRP eligible teaching job on a one-year contract starting in September 2025. You visit family overseas for the summer holidays and return to England to take up your teaching job and apply for the second instalment of the IRP. 

You are eligible for the second instalment of the IRP because you have been continuously employed as a teacher in eligible schools. 
```